client/src/pages/Auth/Login/SignupRedirect.tsx

```diff
@@ -0,0 +1,17 @@
+type SignupRedirectProps = {
+    handleAuthSwitch: () => void
+}
+
+function SignupRedirect({ handleAuthSwitch }: SignupRedirectProps) {
+    return (
+        <div className="bg-blue-400 w-2/5 h-full flex flex-col items-center justify-center">
+            <h2 className="mb-5 font-bold text-3xl">Welcome Back!</h2>
+            <p className="mb-5 mx-10 text-center">To keep connected with us, please signup with your personal info</p>
+            <button className="border-2 border-black px-12 py-1.5 rounded-3xl" onClick={() => handleAuthSwitch()}>
```

Comment:
also seems like this `button` and the styles in it keeps repeating in the app. 

instead of copying the classnames 1 by 1, you might want to create a separate component for this one.
```
function Button({ onClick }: ButtonProps){
 return <button className="border-2 border-black px-12 py-1.5 rounded-3xl" onClick={onClick}>
}
```
 That way you won't have to repeat yourself and just call in `<Button ...>` instead :D And also when adjusting stylings you only have to edit it in 1 place.

Reply:
The same comment ^ also applies to your container, etc.

In fact, you can probably get away creating a separate component for the template. 

Since the `div`, `h2` and `button` are repeating, and the text is the only one changing, you can pass the `text` in each as props instead

Reply:
also gonna leave this link as reference if you want to make it extra clean https://bradfrost.com/blog/post/atomic-web-design/

basically, you structure your files depending on how they're categorized. something like this below

![image](https://user-images.githubusercontent.com/22534285/190416843-d2163678-33e4-4979-bf82-33bcd1ad9c03.png)